SeattleColleges/belindas-closet-nestjs
# Belinda's-Closet-NestJS_3_112_Unit-Tests-Product-Controller-and-Service

Resolves #45 

To test:
Pull this branch and open a terminal in the project
Run the tests for the controller by using the terminal command `npm run test products.controller.spec.ts`
Run the tests for the service by using the command `npm run test products.service.spec.ts`

Resources consulted (there were others, I am bad about closing things when they don't work as expected):
[NestJS docs](https://docs.nestjs.com/fundamentals/testing)
[Article](https://www.tomray.dev/nestjs-unit-testing)
[Article](https://betterprogramming.pub/testing-controllers-in-nestjs-and-mongo-with-jest-63e1b208503c)
[Article](https://blog.logrocket.com/unit-testing-nestjs-applications-with-jest/)
